client-app/modules/push-messages/composables/useWebPushNotifications/useWebPushNotificationsModule.ts

```diff
@@ -15,6 +15,8 @@ import type { FcmSettingsType } from "../../api/graphql/types";
 import type { Messaging } from "firebase/messaging";
 
 const REGISTRATION_SCOPE = "/firebase-cloud-messaging-push-scope";
+const SERVICE_WORKER_VERSION = "1.0.0"; // Have to be updated on every change in the service worker to avoid caching issues
+const SERVICE_WORKER_PATH = "/fcm-service-worker.js";
```

Comment:
I think we should move configuration settings such as SERVICE_WORKER_VERSION and SERVICE_WORKER_PATH to a separate configuration file (e.g., serviceWorkerConfig.ts).

Reply:
@goldenmaya moved them out ot the composable to the module constants file `/constants/index.ts` - do you think it's suitable?